src/app/components/Icons/Arrow.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+
+export function ArrowDown({
+  ...props
+}: React.SVGProps<SVGSVGElement>): JSX.Element {
+  return (
+    <svg
+      className="w-6 h-6"
+      fill="currentColor"
+      viewBox="0 0 20 20"
+      xmlns="http://www.w3.org/2000/svg"
+      {...props}
+    >
+      <path
+        fill-rule="evenodd"
```

Comment:
These kebap-cased 🍢 attributes aren't allowed in JSX, this is probably why your build is failing. Try running `npm run build` locally and see what messages you get